File: SparkApplication/src/main/java/Models/GroupCourse.java
package Models;

public class GroupCourse extends EntityBase{
    public String groupId;
    public String courseId;
    public String teacherId;

    public GroupCourse(){
        super();
    }

    public String getGroupId() {
        return groupId;
    }

    public String getCourseId() {
        return courseId;
    }

    public String getTeacherId() {
        return teacherId;
    }

    public void setGroupId(String groupId) {
        this.groupId = groupId;
    }

    public void setCourseId(String courseId) {
        this.courseId = courseId;
    }

    public void setTeacherId(String teacherId) {
        this.teacherId = teacherId;
    }
}

File: WebApplication/Models/GroupCourse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WebApplication.Models
{
    public class GroupCourse : EntityBase<GroupCourse>
    {
        public string GroupId { get; set; }
        public string CourseId { get; set; }
        public string UserId { get; set; }
    }
}
